tests/test_lookml.py

```diff
@@ -160,21 +159,7 @@ def test_lookml_actual(runner, tmp_path):
     )
     with runner.isolated_filesystem():
         with patch("google.cloud.bigquery.Client", MockClient):
-            result = runner.invoke(
-                lookml,
-                [
-                    "--namespaces",
-                    namespaces.absolute(),
-                ],
-            )
-        sys.stdout.write(result.stdout)
-        if result.stderr_bytes is not None:
-            sys.stderr.write(result.stderr)
-        try:
-            assert result.exit_code == 0
-        except Exception as e:
-            # use exception chaining to expose original traceback
-            raise e from result.exception
+            _lookml(Path(namespaces.absolute()).read_text(), "looker-hub/")
```

Comment:
`namespaces` as provided by `click` should be an open file pointer:
```suggestion
            _lookml(open(namespaces), "looker-hub/")
```

Reply:
I think I tried that... let me commit this suggestion and see